src/components/GoogleAddressLookup/helpers/__test__/getSuggestions.spec.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import getSuggestions from '../getSuggestions';
+import LocationItemIcon from '../../icons/locationItemIcon';
+import SearchValueIcon from '../../icons/searchValueIcon';
+
```

Comment:
here you need to mock getSearchForOption and getFormattedValue and then test that what it is returned by getSearchForOption is concatenated with what it is returned by the map result of calling getFormattedValue for each result.